examples/images_contours_and_fields/interpolation_methods.py

```diff
@@ -34,6 +34,6 @@
 
 for ax, interp_method in zip(axes.flat, methods):
     ax.imshow(grid, interpolation=interp_method, cmap='viridis')
-    ax.set_title(interp_method)
+    ax.set_title(str(interp_method))
```

Comment:
I don't think you need the `str` here.  set_title does that as well...

Reply:
When I looked at the rendered page, the `None` plot has no title, but in the current docs, is does have a title that says "None"

Reply:
Oh, yes, I guess `None` might have a different meaning.  Fair enough